core/common/src/main/java/alluxio/PropertyKey.java

```diff
@@ -19,8 +19,14 @@
  * you change or add Alluxio configuration properties.
  */
 public enum PropertyKey {
+  CONF_DIRECTORY(Name.CONF_DIRECTORY, "${alluxio.home}/conf"),
   DEBUG(Name.DEBUG, false),
+  /**
+   * @deprecated use INSTALL_HOME or WORK_DIRECTORY instead
```

Comment:
the javadoc is on the enum value, so would it makes sense to be `PropertyKey.INSTALL_HOME` and `PropertyKey.WORK_DIRECTORY`?